custom_components/thermal_comfort/const.py

```diff
@@ -0,0 +1,24 @@
+from homeassistant.const import Platform
+from homeassistant.backports.enum import StrEnum
+
+DOMAIN = "thermal_comfort"
+PLATFORMS = [Platform.SENSOR]
+CONF_TEMPERATURE_SENSOR = "temperature_sensor"
+CONF_HUMIDITY_SENSOR = "humidity_sensor"
+CONF_POLL = "poll"
+
+DEFAULT_NAME = "Thermal comfort"
+UPDATE_LISTENER = "update_listener"
+
+
+class SensorType(StrEnum):
+    """Sensor type enum."""
+
+    ABSOLUTEHUMIDITY = "absolutehumidity"
+    DEWPOINT = "dewpoint"
+    FROSTPOINT = "frostpoint"
+    FROSTRISK = "frostrisk"
+    HEATINDEX = "heatindex"
+    SIMMERINDEX = "simmerindex"
+    SIMMERZONE = "simmerzone"
+    THERMALPERCEPTION = "perception"
```

Comment:
Not sure if we should move this to general constants.

Reply:
Returned to sensor.